### PR TITLE
Update build script

### PR DESCRIPTION
**Pull request by stevemessick**
Fix a few problems:
- JxBrowser files were not available to unit tests
- The script wasn't getting updated with the proper IDE and plugin versions
- The result of the external build was not being returned, causing broken bots to be green

### CODE REVIEW

File: tool/plugin/lib/plugin.dart [hunk 1]
```diff
@@ -387,9 +411,9 @@ class GradleBuildCommand extends BuildCommand {
       return await super.doit();
     } finally {
       if (Platform.isWindows) {
-        return await exec('.\\gradlew.bat', ['--stop']);
+        await exec('.\\gradlew.bat', ['--stop']);
```
Comment (jacob314) [inline tool/plugin/lib/plugin.dart:414]: wow. should we have a lint about return statements in finally clauses? @pq @munificent 
Determining which return statement should actually apply in this case is a bit confusing.

Comment (stevemessick) [reply tool/plugin/lib/plugin.dart:414]: I like that idea. I can't imagine why I typed "return" in the first place but I certainly didn't want it. Note the final statement in the protected block is a return, but you might not be able to determine that all the time.

Thanks for the review! Looks like I have a problem in the build script, which I'll fix.

**Comment by stevemessick** [comment]
I had to make so many changes that another review is probably warranted. However, I want to get the patch out.

TBR

File: flutter-idea/build.gradle [hunk 2]
```diff
@@ -33,6 +33,8 @@ intellij {
 dependencies {
   testCompile "org.powermock:powermock-api-mockito2:2.0.0"
   testCompile "org.powermock:powermock-module-junit4:2.0.0"
+  implementation fileTree(dir: "$project.rootDir/lib/jxbrowser",
```
Comment (helin24) [inline flutter-idea/build.gradle:36]: Is this change leftover from working on jxbrowser stuff? Or is it necessary for dealing with the jxbrowser files that are in your directory but not committed?

**Comment by stevemessick** [comment]
That will make the unit test bot in your PR turn green. Just restart it.

**Comment by helin24** [comment]
> That will make the unit test bot in your PR turn green. Just restart it.

Oh ha! Magic! I'll try it.

**Pull request merged**